index.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css">
+        <title>Capstone</title>
+    </head>
+    <body class="bodyart">
+        <header id="header" class="header">
+            <nav class="navbar">
+                <ul class="nav-list">
+                    <li>
+                        <img src="./images/Rectangle.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <img src="./images/twitter.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">English</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">My Page</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">Logout</a>
+                    </li>
+                </ul>
```

Comment:
Nice work so far with your project 👍🏿  but I notice your page only has one page which is the index as a project requirement you are to create two pages for this project the index page and the about us page please do well to create the about us page and all its corresponding layout.
- [ ] Also in the new page about us page please add the correct navigation to be about to go back and forth of the home and About pages

Reply:
About Us was created successfully as you can see in the following image:

![image](https://github.com/jovasal/Capstone/assets/35693330/9a98149b-49ee-4131-94ac-949ae4e70fc9)